spec/models/partner_spec.rb

```diff
@@ -17,10 +17,18 @@
   it { is_expected.to belong_to :province }
   it { is_expected.to belong_to :status }
 
-  it { is_expected.to allow_value(Date.current, Date.yesterday).for :start_date }
-  it { is_expected.not_to allow_value(Date.tomorrow).for :start_date }
-  [7, 'yes', true].each do |bad_date_value|
-    it { is_expected.to_not allow_value(bad_date_value).for :start_date }
+  context 'start_date validation' do
+    describe 'valid date' do
+      it { is_expected.to have_validation :valid_date_presence, :on => :start_date }
+    end
+
+    describe 'not in future' do
+      it { is_expected.to_not allow_value("2011-04-04").for :start_date }
```

Comment:
As above, I think that replacing this with `it { is_expected.to have_validation :date_not_in_future, :on => :father_date_of_death }` is more appropriate. Do you think these three `start_date` validations can be done without the individual `describe` blocks?